src/App.jsx

```diff
@@ -43,6 +44,7 @@ const App = (props) => {
         <Route exact path="/" component={LoginForm} />
         <Route path="/write" component={CreateArticle} />
         <Route path="/review" component={Review} />
+        <Route path="/article/:id" component={PublishArticle} />
```

Comment:
The article publish page will also double as the review page, as per lo-fis. This should preferable by under /review/:id if anything, instead of a separate path.

Reply:
Fixed